packages/theme-chalk/src/checkbox-button.scss

```diff
@@ -125,6 +125,12 @@
     }
   }
 
+  &:only-child {
+    .#{$namespace}-checkbox-button__inner {
+      border-radius: getCssVar('border-radius-base');
+    }
+  }
+
```

Comment:
Maybe split the `:first-child` and `:last-child` border-radius corners instead of overriding the style.

``` diff
- border-radius: getCssVar('border-radius-base') 0 0
-        getCssVar('border-radius-base');
+ border-top-left-radius:  getCssVar('border-radius-base');
+ border-bottom-left-radius:  getCssVar('border-radius-base');
```